pkg/controller/efs/filesystem/setup.go

```diff
@@ -126,3 +152,60 @@ func postCreate(_ context.Context, cr *svcapitypes.FileSystem, obj *svcsdk.FileS
 	meta.SetExternalName(cr, pointer.StringValue(obj.FileSystemId))
 	return managed.ExternalCreation{}, nil
 }
+
+func (e *custom) cacheBackupPolicy(filesystemId *string) error {
+	// Get BackupPolicy
+	// default is no backup
+	policy, err := e.client.DescribeBackupPolicy(&svcsdk.DescribeBackupPolicyInput{FileSystemId: filesystemId})
+	if err != nil {
+		e.cache.backupPolicy = !backupPolicyIsNotFound(err)
+		return resource.Ignore(backupPolicyIsNotFound, err)
+	}
+	policyStatus := aws.StringValue(policy.BackupPolicy.Status)
+	switch policyStatus {
+	case "ENABLED", "ENABLING":
+		e.cache.backupPolicy = true
+	case "DISABLED", "DISABLING":
+		e.cache.backupPolicy = false
+	}
+	return nil
+}
+
+func (e *custom) updateBackupPolicy(ctx context.Context, cr *svcapitypes.FileSystem, obj *svcsdk.DescribeFileSystemsOutput) error {
+	for _, res := range obj.FileSystems {
+		err := e.cacheBackupPolicy(res.FileSystemId)
```

Comment:
This `cache` thingy is only used as a workaround to pass state information from `Observe` to `Create`, `Update` or `Delete` so you don't have to recompute it, i.e. difference in resource tags - because the controller framwork does not allow passing custom values in a `managed.ExternalObservation`.